CHANGELOG.md

```diff
@@ -69,8 +69,7 @@ The change log is available [on GitHub][2].
 * [#58](https://github.com/kowainik/tomland/issues/58):
   Add `decodeFile` function.
 
-0.3
-=====
+## 0.3
```

Comment:
```suggestion
## 0.3.0
```

Reply:
It's actually `0.3` on Hackage (for some unknown reason):

* http://hackage.haskell.org/package/tomland-0.3

Reply:
That's a tragedy!

Reply:
Super minor, and I don't know why GitHub won't let me comment on that line, but I think two empty lines (line 88 and 92, above "_Migration guide") make the section 0.3 look different than the other sections.

Reply:
@jiegillet Yeah, in GitHub you can comment only to diff lines or some near lines (which is quite sad, because sometimes people might not be aware of some distant code.

Regarding empty lines: this is needed for showing `Migration guide` on separate line. See the rendered CHANGELOG. I've just discussed with @vrom911 how we want to show breaking changes. Will do this now.

Reply:
Yes, this is because the rendered file should show it on the new line, and we need to add extra newlines for that in markdown. 
But good point, thank for paying attention to such details! 👍